backends/platform/atari/readme.txt

```diff
@@ -0,0 +1,414 @@
+ScummVM
+=======
+
+This is a new port of ScummVM (https://www.scummvm.org), a program which allows
+you to run certain classic graphical adventure and role-playing games, provided
+you already have their data files.
+
+You can find a full list with details on which games are supported and how well
+on the compatibility page: https://www.scummvm.org/compatibility.
+
+
+Yet another port?
+-----------------
+
+Yes, I am aware of the official Atari/FreeMiNT port done by KeithS over the
+years (https://docs.scummvm.org/en/v2.6.1/other_platforms/atari.html). It is
+even updated every release and put on the official ScummVM website. That port
+is basically just a recompiled SDL backend for our platform - that has some
+advantages (works in GEM, can be easily compiled for the FireBee etc) but also
+a few disadvantages, like:
+
+- It's huge, massively huge, about 51 MiB. It contains about every engine,
```

Comment:
Using dynamic plugins, either using LDG or by extending our built-in ELF loader, would have a significant benefit here in reducing memory usage without having to disable many engines.

Reply:
Are you referring to [Atari LDG](https://docs.google.com/document/d/1d8MSFrcVLW_bLnO7bhokuOTNXoM6yK2pQ8O5oyPx9oI/edit) ? Hmm, that is an interesting idea, for some reason it hasn't occurred to me at all. Definitely worth considering, added into readme.txt.

ELF is not available on our platform, unfortunately.

Btw, I don't plan to enable the 16bpp engines if when this is done, too much data to handle/update on our poor platform. But reducing the executable to some reasonable size would be great, ideally when being able to get rid of all the theme support code with it.